daphne/src/lib.rs

```diff
@@ -225,6 +236,25 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// Global DAP parameters common across tasks.
+//
+// TODO spec: `max_window_size`, `min_duration_start`, and `max_duration_end`
```

Comment:
These need to be renamed to match struct.

Reply:
Thanks for pointing this out, it seemed that there were other locations that used the legacy variable name, so I've updated those in addition to this.